bench

```diff
@@ -65,7 +65,14 @@ mkdir -p $VDIR1 $VDIR2
 
 touch $IFILE
 
-ARCH=`arch -k`
+DISTRO=$(cat /etc/issue | head -n +1 | awk '{print $1}')
```

Comment:
Can you change this to, `cat /etc/issue 2> /dev/null`, so that on distributions where `/etc/issue` does not exist, the error message won't be emitted?